contracts/interfaces/ILineOfCredit.sol

```diff
@@ -30,37 +30,43 @@ interface ILineOfCredit {
     address indexed lender,
     address indexed token,
     uint256 indexed deposit,
-    bytes32 positionId
+    bytes32 id
   );
-  // can reference only id once AddCredit is emitted because it will be indexed offchain
+  // can only reference id once AddCredit is emitted because it will be indexed offchain
 
-  event SetRates(bytes32 indexed id, uint128 indexed drawnRate, uint128 indexed facilityRate);
+  event SetRates(bytes32 indexed id, uint128 indexed dRate, uint128 indexed fRate);
 
   event IncreaseCredit (bytes32 indexed id, uint256 indexed deposit);
 
   // Lender Events
 
   event WithdrawDeposit(bytes32 indexed id, uint256 indexed amount);
-  // lender removing funds from Line  principal
+  // Emits data re Lender removes funds (principal) - there is no corresponding function, just withdraw()
```

Comment:
Can u make all the comments *above* the thing that is being documented plz. So ove all the comments on top of the event definitions

Reply:
done in pr 147